### PR TITLE
DOCS-11860 initial commit for pass-through docs (plus other edits)

### PR DESCRIPTION
**Pull request by IsaacEldridge**
@mauro-dentoni this is the PR for pass-through auth. there are a few other changes unrelated to pass-through so I left a comment on the relevant places to check.

### CODE REVIEW

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 1]
```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
 ** *I already have credentials...*: Click *Next*, provide an existing client ID and secret pair, and then click *Authenticate & Request Access*.
 . To change your access method from the *Run Query* page, click *Manage Credentials* > *Switch credentials*.
 
-=== Request Access to Environments Using an External Client Provider
+
+== Write and Run a Query
+
+Use the query editor to write, run, and troubleshoot queries. 
+
+. If any of the APIs in the unified schema require pass-through authentication, click *Add HTTP Headers*, and:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/query-unified-schema.adoc:33]: @mauro-dentoni this is for adding pass-through auth before running a query

File: modules/ROOT/pages/supported-authentication-methods.adoc [hunk 2]
```diff
@@ -30,9 +30,19 @@ curl GET 'http://localhost/myResource' \
   "password": "myPassword"
 }'
 --
+
+* *Pass-through authentication*
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/supported-authentication-methods.adoc:34]: @mauro-dentoni this is for pass-through auth docs on the supported authentication methods page

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 3]
```diff
@@ -50,6 +49,7 @@ The first task to adding an API to the unified schema is adding the API to Anypo
 +
 * *No Auth*: Use if your API is public.
 * *Basic Auth*: Use if you authenticate to your API using a username and password.
+* *Pass-through*: Use if you authenticate to your API by passing authorization headers.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:52]: @mauro-dentoni this is for adding pass-through auth when you add an api to the unified schema

File: modules/ROOT/pages/promote-api.adoc [hunk 4]
```diff
@@ -52,20 +52,12 @@ To promote an API:
 . Click *Next: Provide Authentication*.
 . On the *Provide authentication* page, select the type of authentication on the GET endpoints for the API that you want to add, allowing Anypoint DataGraph to make requests to your API when queried:
 +
---
 * *No Auth*: Use if your API is public.
 * *Basic Auth*: Use if you authenticate to your API using a username and password.
+* *Pass-through*: Use if you authenticate to your API by passing authorization headers.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/promote-api.adoc:57]: @mauro-dentoni this is for pass-through auth when you promote an api

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 5]
```diff
@@ -105,6 +106,16 @@ If you need to edit the URL of an API that you've added to the unified schema, y
 . Type in the new URL.
 . Click *Save*.
 
+[[change-the-authentication-method-for-an-api-aded-to-the-unified-schema]]
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:109]: @mauro-dentoni these are the steps to change auth method when you edit an API

File: modules/ROOT/pages/query-unified-schema.adoc (hunk 1 above)
Comment (mauro-dentoni) [reply modules/ROOT/pages/query-unified-schema.adoc:33]: The statement looks fine Isaac. We will enforce the add of the headers, before running a query, if any of the APIs requires pass-through authentication. Thanks

File: modules/ROOT/pages/supported-authentication-methods.adoc (hunk 2 above)
Comment (mauro-dentoni) [reply modules/ROOT/pages/supported-authentication-methods.adoc:34]: Looks fine

File: modules/ROOT/pages/add-api-to-unified-schema.adoc (hunk 3 above)
Comment (mauro-dentoni) [reply modules/ROOT/pages/add-api-to-unified-schema.adoc:52]: Looks fine

File: modules/ROOT/pages/promote-api.adoc (hunk 4 above)
Comment (mauro-dentoni) [reply modules/ROOT/pages/promote-api.adoc:57]: Looks fine

File: modules/ROOT/pages/add-api-to-unified-schema.adoc (hunk 5 above)
Comment (mauro-dentoni) [reply modules/ROOT/pages/add-api-to-unified-schema.adoc:109]: Looks fine. With the current behavior, the only way of editing the configuration of an authentication of an API is by removing the configuration and adding a new one.

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 6]
```diff
@@ -105,6 +106,16 @@ If you need to edit the URL of an API that you've added to the unified schema, y
...
+== Change the Authentication Method for an API Added to the Unified Schema
+
+If you need to edit the authentication method of an API that you've added to the unified schema, you can do so on the *API details* page.
+
+. Click *List of APIs added* and select the appropriate API.
+. Click *API details*.
+. In the *Authentication* panel, select an authentication policy and complete any necessary fields.
```
Comment (mauro-dentoni) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:116]: @IsaacEldridge, please, review if the functionality works as you described. As far as I've checked, if you have added an authentication method previously, when changing it, you should first remove the old configuration (Button "Remove and add a new one") and then add the new one. I'd turn the statement like this (or similar): "In the *Authentication* panel, remove the old authentication, select the new authentication policy and complete any necessary fields."

File: modules/ROOT/pages/promote-api.adoc [hunk 7]
```diff
@@ -52,20 +52,12 @@ To promote an API:
...
 * *Client ID enforcement via headers*: Use if you authenticate to your API passing `client_id` and `client_secret` headers.
 * *Client ID enforcement via query parameters*: Use if you authenticate to your API passing `client_id` and `client_secret` query params.
 * *Custom*: Use if you authenticate to your API using custom header parameters and values.
---
-+
-[IMPORTANT]
---
-Any other authentication methods to access your API, such as custom OAuth implementations or short-lived token methods, are not supported.
```
Comment (mauro-dentoni) [inline modules/ROOT/pages/promote-api.adoc:65]: Some contextual information for you @IsaacEldridge ... we are planning to implement OAuth support by End-Q4 (To be confirmed this week)

Comment (IsaacEldridge) [reply modules/ROOT/pages/promote-api.adoc:65]: got it!

**Review by mauro-dentoni — APPROVED** [review]
Thanks for adjusting the authentications modification instructions. Everything looks fine now. I'm approving these changes

**Review by emileeburkeVT — DISMISSED** [review]

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 8]
```diff
@@ -105,6 +106,19 @@ If you need to edit the URL of an API that you've added to the unified schema, y
 . Type in the new URL.
 . Click *Save*.
 
+[[change-the-authentication-method-for-an-api-aded-to-the-unified-schema]]
```
Comment (hannanelson) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:109]: "aded" ok?

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 9]
```diff
@@ -105,6 +106,19 @@ If you need to edit the URL of an API that you've added to the unified schema, y
...
+== Change the Authentication Method for an API Added to the Unified Schema
+
+If you need to edit the authentication method of an API that you've added to the unified schema, you can do so on the *API details* page. You must remove the current authentication method and add a new one.
+
+To edit the authentication method:
+
+. Click *List of APIs added* and select the appropriate API.
+. Click *API details*.
+. In the *Authentication* panel, *Click remove and add a new one*.
```
Comment (hannanelson) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:118]: Is this correct?
*Click remove and add a new one*.

File: modules/ROOT/pages/edit-schema.adoc [hunk 10]
```diff
@@ -11,17 +11,27 @@ To edit an API schema, address the following tasks where applicable:
 If there are types that have a conflict with the unified schema, make sure to resolve them all. See xref:resolve-conflicts.adoc[] for more information.
 * Enable collaboration on an object type
 +
-Enable collaboration for applicable types and provide the required settings. This step is optional, but when you enable collaboration on types, you'll create a more connected and enriched unified schema. See xref:collaboration.adoc[] for more information.
+Enable collaboration for applicable types and provide the required settings. This step is optional, but when you enable collaboration on types, you'll create a more connected and enriched unified schema. Refer to xref:collaboration.adoc[] for more information.
```
Comment (hannanelson) [inline modules/ROOT/pages/edit-schema.adoc:14]: but wwwhhhhyyyy?

File: modules/ROOT/pages/add-api-to-unified-schema.adoc (hunk 8 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:109]:
```suggestion
[[change-the-authentication-method-for-an-api-added-to-the-unified-schema]]
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 11]
```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
...
+
+.. From the *Header Name* drop-down list, select a header name.
```
Comment (hannanelson) [inline modules/ROOT/pages/query-unified-schema.adoc:35]:
```suggestion
.. Select a header name.
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 12]
```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
...
+.. In the *Header Value* field, add a header value.
```
Comment (hannanelson) [inline modules/ROOT/pages/query-unified-schema.adoc:36]:
```suggestion
.. Enter a header value.
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 13]
```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
...
+.. To specify additional header name and value pairs for other APIs, click *+Add Header* x 
+.. Click *Save*.
+.. To edit the HTTP headers, click *Edit HTTP Headers*.
+
+. In the query editor, start typing your query using the in-line autocomplete function or type *Option/Alt + Shift + Space* to open the autocomplete menu.
```
Comment (hannanelson) [inline modules/ROOT/pages/query-unified-schema.adoc:41]: Do we use font changes for key combos? I can't remember.

File: modules/ROOT/pages/add-api-to-unified-schema.adoc (hunk 8 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/add-api-to-unified-schema.adoc]: fixed

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 14]
```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
...
+.. Enter a header value.
+.. To specify additional header name and value pairs for other APIs, click *+Add Header* x 
+.. Click *Save*.
+.. To edit the HTTP headers, click *Edit HTTP Headers*.
+
+. In the query editor, start typing your query using the in-line autocomplete function or type *Option/Alt + Shift + Space* to open the autocomplete menu.
++
+image::datagraph-qsg-query-autocompletion.png[Using inline autocompletion to write a query,70%]
+
+. Optionally, in the *Query Variables* pane, add any query variables you want to use.
```
Comment (hannanelson) [inline modules/ROOT/pages/query-unified-schema.adoc:45]: Kim frowns upon "optional"

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 15]
```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
...
+* Time taken by Anypoint DataGraph to parse and validate the query
+* Total response time for the entire query
+* Duration of requests to each source API in the query
+
+. To see response logs associated with the query, click *View response logs*.
++
+image::datagraph-qsg-response-logs.png[Query response logs page]
++
+Log levels for Anypoint DataGraph include, DEBUG, INFO, WARN, and ERROR.
+. Click *View History* to access this same query—or others—at a later time.
```
Comment (hannanelson) [inline modules/ROOT/pages/query-unified-schema.adoc:67]: This is weird:
query—or others—at

I think Kim frowns upon dashes used like this, too. But, if you keep them, I think they need spaces.

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 16]
```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
...
+. To use this query in your application, click *Copy endpoint* to copy the query and the automatically generated GraphQL endpoint. You can also copy the query as a cURL snippet.
+
+== Query Editor Shortcuts
+The following table provides shortcuts you can use in the query editor:
```
Comment (hannanelson) [inline modules/ROOT/pages/query-unified-schema.adoc:71]: I think you need a blank line here - does this build correctly?

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 17]
```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
...
+
+[%header%autowidth.spread]
+|===
```
Comment (hannanelson) [inline modules/ROOT/pages/query-unified-schema.adoc:74]: Do we use lowercase for keystroke combos?

**Review by hannanelson — APPROVED** [review]

**Pull request merged**